src/loader/mod.rs

```diff
@@ -53,7 +59,14 @@ pub enum Error {
     Elf(elf::Error),
 
     /// Failed to load PE image.
-    #[cfg(all(feature = "pe", target_arch = "aarch64"))]
+    #[cfg(all(
+        feature = "pe",
+        any(
+            target_arch = "aarch64",
+            target_arch = "riscv32",
```

Comment:
Agreed, unless for whatever reason `riscv32` gains popularity in virtualization industry, I can't justify the worth of maintaining `riscv32` support at this stage.

For `kvm-bindings`, however, I think there is no such concern, since the same Linux header is applicable to both `riscv32` and `riscv64`.